practice-two/src/components/common/Link.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Link Component
+ * @param color - The color of the link.
+ * @param disabled - Determines if the link is disabled or not.
+ * @param level - The level of the link's text. Possible values: 'body3', 'body2'.
+ * @param underline - Determines if the link text should be underlined or not.
+ * @param variant - Additional CSS classes to customize the link's appearance.
+ * @param children - The content to be displayed inside the link.
+ * @returns {JSX.Element} The Link component.
+ */
+function Link({ color, disabled, level, underline, variant, children }) {
+  const linkStyles = {
+    textDecoration: underline ? 'underline' : 'none',
+    color: disabled && color,
```

Comment:
Please check: `color: disabled && color`

Reply:
if yes, I don't think it's correct since I used color: 'red' but only disabled this component to see the 'red'!